src/pages/codingClubs/CodingClubs.tsx

```diff
@@ -0,0 +1,99 @@
+import { type FC } from "react"
+import { Typography, useTheme } from "@mui/material"
+import { Download as DownloadIcon } from "@mui/icons-material"
+
+import * as page from "codeforlife/components/page"
+
+import CodeClubHeroImage from "../../images/coding_club_hero_hexagon.png"
+import AboutUsImage from "../../images/about_us.jpg"
+import PythonClubImage from "../../images/coding_club_python_pack.png"
+import Introduction from "../../components/Introduction"
+
+import ClubAim from "./ClubAim"
+import { Link, LinkButton } from "codeforlife/components/router"
+
+const CodingClubs: FC = () => {
```

Comment:
General rule. If we export a component, you must always export the props for the component. Even if the props are empty.

```tsx
export interface CodingClubsProps {}

const CodingClubs: FC<CodingClubsProps> = () => {...}

export default CodingClubs
```